kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/outputs/parquet/ParquetOutputFiles.java

```diff
@@ -99,7 +99,7 @@ public void writeOutputTables(Map<String,VariablesMap> metadataVariables) throws
 		            for (GenericData.Record recordData : dataset) {
 		                writer.write(recordData);
 		            }
-		            log.info("Parquet datasize for {} is : {} for {} records ",datasetName,writer.getDataSize(), dataset.size());
+		            log.info("Parquet datasize for {} is : {} for {} records ",datasetName,writer.getDataSize(), dataset == null ? 0 : dataset.size());
```

Comment:
maybe add a log.warn before this if the dataset is null? (there may be a code smells here if the dataset can be null)